lib/linked_in/refresh_token.rb

```diff
@@ -0,0 +1,12 @@
+module LinkedIn
+  # Refresh Token API
+  # @see https://learn.microsoft.com/en-us/linkedin/shared/authentication/programmatic-refresh-tokens
+  class RefreshToken < APIResource
+
+    def refresh_token(options = {})
+      path = 'oauth/v2/accessToken'
+      post(path, options, 'Content-Type' => 'application/x-www-form-urlencoded')
```

Comment:
it this fine to skip as already define [here](https://github.com/marlytics/linkedin-v2/pull/1/files#diff-b3ebbc1e250e18d9f944c4bf9dc292f41bc1206b0abff4bcf319515b97ccc507R19)? 🤔 
```suggestion
      post(path, options)
```

Reply:
Since we are using default_headers while initializing the connection and default_headers have `"Content-Type" => "application/json"` . So I guess therefore passing the content type separately from here was needed.